### DISCUSSION TITLE
How to create a new cluster in Nightingale

### DISCUSSION THREAD

**Discussion by wangxiajian** [ccfos/nightingale#1213] [General]
您好，我想请问一下夜莺该如何创建新的集群
![image](https://user-images.githubusercontent.com/67364083/196864756-31945171-d262-4b12-b3ef-2d132f892549.png)

**Comment by UlricQin** [comment]
https://mp.weixin.qq.com/s/rJ3ixO2EhXIbQD0PTT_Bng
